.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: "CodeQL config"
+
+paths-ignore:
+  - '.pnp.cjs'
```

Comment:
Otherwise you get a false positive for this generated file:

![image](https://user-images.githubusercontent.com/21194782/222395604-ccba8917-ee78-4a2f-b78d-3fa5c5c787f6.png)